static/js/publisher/release/components/releasesTable/releaseCell.js

```diff
@@ -92,6 +92,7 @@ const ReleasesTableReleaseCell = (props) => {
   };
 
   function handleHistoryIconClick(arch, risk, track, branchName) {
+    window.scrollTo(0, 0);
```

Comment:
I always think when we do this we should store where the user came from (scroll) so when they close we can send them back - not for now, just to think about.